src/util/weak-map.js

```diff
@@ -22,6 +22,7 @@ export default window.WeakMap || (function () {
           Object.defineProperty(obj, key, {
             configurable: true,
             enumerable: false,
+            writable: true,
```

Comment:
This was causing mappings to fail in Opera 12 where it simply would no-op if you didn't set this to `true`. Other browsers seemed to work probably because they already have `WeakMap`.